app-information.html

```diff
@@ -0,0 +1,369 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Web App Manifest - App Information
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class='remove'>
+    var respecConfig = {
+      mdn: true,
```

Comment:
```suggestion
```

Reply:
We still would like this on MDN as web devs are using it

Reply:
Yes, if there are MDN pages associated with these members then it's prudent to show them. They will be displayed as not having any browser support tho, but I guess that's ok.